graphql/resolvers.go

```diff
@@ -863,6 +863,15 @@ func (r *queryResolver) AwsRegions(ctx context.Context) ([]string, error) {
 	return regions, nil
 }
 
+func (r *queryResolver) SiteBanner(ctx context.Context) (*SiteBanner, error) {
+	settings := evergreen.GetEnvironment().Settings()
```

Comment:
I just noticed this field only changes when i restart the local evergreen server. Is there a better place to fetch this info from?

Reply:
That field is intentionally cached for performance. To get the setting in real-time, use the evergreen.GetConfig function